config.json

```diff
@@ -470,6 +470,15 @@
       ]
     },
     {
+      "slug": "minesweeper",
+      "difficulty": 7,
+      "topics": [
```

Comment:
Very basic ones (probably too basic, I'm not sure if it should be included in an exercise with difficulty 7):
- Control-flow (conditionals)
- Control-flow (loops)

This one might not describe the exercise, but as it can be implemented with an array of arrays...
- Matrices